cli/003-cli-autocomplete.md

```diff
@@ -0,0 +1,29 @@
+- Start Date: 2021-06-11
+- RFC PR:
+- Issue:
+
+### Commands auto-completion
+Support auto-completion of commands in bash. Lesser priority - zsh.
```

Comment:
There are libraries that can generate shell completions like https://click.palletsprojects.com/en/8.0.x/shell-completion/.
Maybe one exists for golang too.